buddyboss/buddyboss-platform
# PROD-895 PROD-2512 Fixed #1787 Comma operator issues with members count

### Changes proposed in this Pull Request:



Fixes #1787 

### How to test the changes in this Pull Request:

1. Import or create 1K plus members.
2. Go to "Members" Page.
3. See All Members count. 

### Proof Screenshots or Video


Screenshot : https://prnt.sc/124xdle
